eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -99,5 +99,17 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 }
 
 func (c *ClusterLoader) buildArgs() string {
-	return fmt.Sprintf("[%s]", strings.Join(c.Config.Spec.ClusterLoader.TestParams, ","))
+	return fmt.Sprintf("[%s]", strings.Join(append(c.Config.Spec.ClusterLoader.TestParams, "--report-dir=/usr/share/cluster-loader"), ","))
+}
+
+func (c *ClusterLoader) buildS3Command() string {
+	// Separating large bash command for readability
+	bashVars := "sizeoffiles=$(ls /var/reports/cluster-loader -l | /bin/bash -c 'wc -l') ; initial=$sizeoffiles"
+
+	// have condition be add 2 because only keeping track of 2 metrics in my tests, so directory will be at least 2 files larger after test
```

Comment:
If I understand it, this variable will change if we keep track of multiple metrics (i.e. multiple files). Let's chat offline to brainstorm ways that we can determine if CL2 is done or not.

Reply:
Synced offline, will use initContainers instead.